xrspatial/aspect.py

```diff
@@ -183,113 +196,90 @@ def aspect(agg: xr.DataArray,
-
+        >>> import numpy as np
+        >>> import xarray as xr
+        >>> from xrspatial import aspect
+
+        >>> data = np.array([
+            [1, 1, 1, 1, 1],
+            [1, 1, 1, 2, 0],
+            [1, 1, 1, 0, 0],
+            [4, 4, 9, 2, 4],
+            [1, 5, 0, 1, 4],
+            [1, 5, 0, 5, 5]
+        ], dtype=np.float64)
+        >>> raster = xr.DataArray(data, dims=['y', 'x'], name='raster')
+        >>> print(raster)
+        <xarray.DataArray 'raster' (y: 6, x: 5)>
+        array([[1., 1., 1., 1., 1.],
+               [1., 1., 1., 2., 0.],
+               [1., 1., 1., 0., 0.],
+               [4., 4., 9., 2., 4.],
+               [1., 5., 0., 1., 4.],
+               [1., 5., 0., 5., 5.]])
+        Dimensions without coordinates: y, x
+        >>> aspect_agg = aspect(raster)
+        >>> print(aspect_agg)
+        <xarray.DataArray 'aspect' (y: 6, x: 5)>
+        array([[ nan,  nan,  nan,  nan, nan],
+               [ nan,  -1.        ,   225.      ,   135.      , nan],
+               [ nan, 343.61045967,   8.97262661,  33.69006753, nan],
+               [ nan, 307.87498365,  71.56505118,  54.46232221, nan],
+               [ nan, 191.30993247, 144.46232221, 255.96375653, nan],
+               [ nan,  nan        ,  nan        ,  nan        , nan]])
+        Dimensions without coordinates: y, x
+
+    Aspect works with Dask with NumPy backed xarray DataArray
     .. sourcecode:: python
-
-        >>> print(aspect_agg[200:203, 200:202])
-        <xarray.DataArray 'Aspect' (lat: 3, lon: 2)>
-        array([[155.07530658, 146.26526699],
-               [194.81685088, 136.55836607],
-               [203.14170549, 187.97760934]])
-        Coordinates:
-        * lon      (lon) float64 -9.975e+06 -9.925e+06
-        * lat      (lat) float64 -6.633e+06 -6.567e+06 -6.5e+06
-        Attributes:
-            res:            (50000.0, 66666.66666666667)
-            Description:    Example Aspect
-            units:          deg
-            Max Elevation:  4000
+        >>> import dask.array as da
+        >>> data_da = da.from_array(data, chunks=(3, 3))
+        >>> raster_da = xr.DataArray(data_da, dims=['y', 'x'], name='raster_da')
+        >>> print(raster_da)
+        <xarray.DataArray 'raster' (y: 6, x: 5)>
+        dask.array<array, shape=(6, 5), dtype=int64, chunksize=(3, 3), chunktype=numpy.ndarray>
+        Dimensions without coordinates: y, x
+        >>> aspect_da = aspect(raster_da)
+        >>> print(aspect_da)
+        <xarray.DataArray 'aspect' (y: 6, x: 5)>
+        dask.array<_trim, shape=(6, 5), dtype=float64, chunksize=(3, 3), chunktype=numpy.ndarray>
+        Dimensions without coordinates: y, x
+        >>> aspect_da.compute()  # compute the results
+        <xarray.DataArray 'aspect' (y: 5, x: 5)>
+        array([[nan,     nan,        nan,        nan,          nan],
+               [nan, 120.96375653, 104.03624347, 135.        , nan],
+               [nan, 101.30993247, 108.43494882, 123.69006753, nan],
```

Comment:
I get values similar to the `numpy` values above. Maybe this is a copy and paste error?

Reply:
Thanks for catching this, just fixed!